### PR TITLE
Js tabloid-userposts

### PR DESCRIPTION
**Pull request by jsweeten**
Added User Post feature which allows the current logged in user to view their own posts
Updated Navbar to include link for user navigation

### CODE REVIEW

**Comment by jsweeten** [comment]
closes #44

File: Tabloid/Controllers/PostController.cs [hunk 1]
```diff
@@ -20,7 +20,11 @@ public IActionResult Get()
         {
             return Ok(_postRepository.GetAll());
         }
-
+        
+        [HttpGet("{firebaseUserId}")]
```
Comment (joshdbarton) [inline Tabloid/Controllers/PostController.cs:24]: This method should be auth protected.

Comment (joshdbarton) [reply Tabloid/Controllers/PostController.cs:24]: you should get the fbUserId here, out of the token, rather than passing the fbUserId in the url.

Comment (jsweeten) [reply Tabloid/Controllers/PostController.cs:24]: Took care of it. Thanks!

**Review by joshdbarton — APPROVED** [review]
:shipit:

**Pull request merged**